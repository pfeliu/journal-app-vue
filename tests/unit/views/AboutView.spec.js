import { shallowMount } from '@vue/test-utils';
import AboutView from '@/views/AboutView.vue';

describe('About View ', () => {
  let wrapper;

  wrapper = shallowMount(AboutView);

  test('should render About view', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
