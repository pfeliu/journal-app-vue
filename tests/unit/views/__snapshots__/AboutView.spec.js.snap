// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About View  should render About view 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
